tardis/atomic.py

```diff
@@ -19,15 +19,13 @@ class AtomDataNotPreparedError(Exception):
 
 logger = logging.getLogger(__name__)
 
-default_atom_h5_path = os.path.join(os.path.dirname(__file__), 'data', 'atom_data.h5')
 
 def data_path(fname):
-    data_dir = os.path.join(os.path.dirname(__file__), 'data')
-    return os.path.join(data_dir, fname)
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', fname)
```

Comment:
I'm trying to look out for PEP8 errors but since I'm not doing a checkout of every PR I have a look at I won't see them (I have a pep checker for my editor only).